tasks/README.md

```diff
@@ -0,0 +1,90 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<end date>",
+  "startedOn":"<start date>",
+  "status": "Active",
+  "ownerId":"<app owner user id>",
+  "percentCompleted":10%,
+  "dependsOn": [
+    <tasksList>
+  ],
+  "participants": [list of users participating in this tasks. For 'group' type only],
+  "completionAward": { gold: 3, bronze: 300 },
+  "lossRate": { gold: 1 }  // Loss per day of overshoot after deadline
+}
+```
+
+## **Requests**
+
+|               Route                |    Description    |
+| :--------------------------------: | :---------------: |
+|      [GET /tasks](#get-tasks)      | Returns all tasks |
+|     [POST /tasks](#post-tasks)     | Creates new task  |
+| [PATCH /tasks/:id](#patch-tasksid) |   Updates tasks   |
+
+## **GET /tasks**
+
+> returns all the tasks
+
+- **Success Response:**
+- **Code:** 200
+  - **Content:** `{ 'message': 'Tasks returned successfully!', 'tasks': [{<task_object>},{<task_object>},] }`
+
+```
+[
+  {<task_object>},
+  {<task_object>}
+]
+
+```
```

Comment:
Thanks for doing the changes! 🎉  @sumitd94 
Just wanted to ask that do we need to include `[  {<task_object>}, {<task_object>} ]` again? I think you have already included it on line number 40. 😄

Reply:
So the reason why I did not remove this is just because to make it visually more clear that we are returning an array of objects. please let me know if that is not needed and will remove it.

Reply:
What she meant was we're already showing it in success response. It's being shown again.

Reply:
I agree with you @sumitd94 . So, how about if we remove it from line number 40 and display it like this -
![image](https://user-images.githubusercontent.com/66248039/103931129-cbe04d00-5145-11eb-9494-842f15f0705d.png)
 What is your opinion @swarajpure ?

Reply:
Yeah this is better
It was being repeated earlier

Reply:
Yeah right. Including it this way will avoid duplication and will also look visually more clear as @sumitd94 said. 😄

Reply:
Cool, Makes sense 👍